src/main/java/frc/robot/Robot.java

```diff
@@ -55,14 +57,15 @@ public void robotInit() {
     // Zero all nesscary sensors on Robot
     ZeroSensors();
 
-    // Reset Robot State
-    // Wherever the Robot is now is the starting position
-    mRobotLogger.log("Robot State Reset");
-    mRobotState.reset();
+    EventWatcherThread.getInstance().start();
 
     // prepare the network table
     NetworkTableInstance inst = NetworkTableInstance.getDefault();
     mTable = inst.getTable("SmartDashboard");
+    // Reset Robot State
+    // Wherever the Robot is now is the starting position
+    mRobotLogger.log("Robot State Reset");
```

Comment:
Should reset() do the logging?

Reply:
moved log message into RobotState.reset()